src/analysis/complex/upper_half_plane/basic.lean

```diff
@@ -269,6 +269,31 @@ lemma denom_apply (g : SL(2, ℤ)) (z : ℍ) : denom g z = (↑g : matrix (fin 2
 
 end SL_modular_action
 
+section upper_half_plane_manifold
+
+/--The upper half space as a subset of `ℂ` which is convenient sometimes.-/
+def upper_half_space := {z : ℂ | 0 <  z.im}
+
+
+lemma hcoe : upper_half_space = coe '' (set.univ : set upper_half_plane) :=
```

Comment:
A better statement might be
```suggestion
lemma upper_half_space_eq_range : upper_half_space = set.range (coe : upper_half_plane \to _) :=
```

Reply:
Sorry I forgot to remove these changes, I don't need any of these things now!